source/includes/_entitytoentityflow.md

```diff
@@ -0,0 +1,84 @@
+# Entity to Entity Flows
+
+For Bitcoin, the currently supported Miners/Exchanges are:
+
+| Name           | `from_entity`    | `to_entity`                                                                           |
+|----------------|------------------|---------------------------------------------------------------------------------------|
+| AntPool        | `antpool`        | `bitstamp`, `bittrex`                                                                 |
+| BTC.TOP        | `btc.top`        | `binance`, `bitfinex`, `bitmex`, `bitstamp`, `bittrex`, `huobi`, `kraken`, `poloniex` |    
+| BitClubNetwork | `bitclubnetwork` | `bitfinex`, `bitstamp`                                                                |
+| F2Pool         | `f2pool`         | `binance`, `bitfinex`, `bitstamp`, `bittrex`, `huobi`, `kraken`, `poloniex`           |
+| SlushPool      | `slushpool`      | `binance`, `bitfinex`, `bitmex`, `bitstamp`, `bittrex`, `huobi`, `kraken`, `poloniex` |
+| viaBTC         | `viabtc`         | `bitmex`, `bitstamp`, `bittrex`, `huobi`, `poloniex`                                  |
+
+The above table defines supported pairs of the query parameters `from_entity` and `to_entity`. 
+
+## Entity to Entity Full Historical Flows
+
+<img src="https://img.shields.io/badge/Tier-Professional-black.svg"/>
+
+This endpoint returns the full historical on-chain entity-to-entity flows.
+
+```shell
+curl "https://api.tokenanalyst.io/analytics/private/v1/entity_to_entity_flow_window_historical/last?window=1h&limit=2&from_entity=antpool&to_entity=bittrex&format=json&token=btc&key=APIKEY"
+```
+
+> The above command returns JSON structured like this:
+
+```json
+[
```

Comment:
change this data to instances where there are actual values (not 0's across the board)

And update the API link in 'curl "https://api.tokenanalyst.io/analytics/private/v1/entity_to_entity_flow_window_historical/last?window=1h&limit=2&from_entity=antpool&to_entity=bittrex&format=json&token=btc&key=APIKEY"
' to reflect where you got the data - so user can copy paste and get similar results